### DISCUSSION TITLE
Location of the Blockchain db

### DISCUSSION THREAD

**Discussion by hibenji** [Chia-Network/chia-blockchain#6436] [Support]
I have a linux machine that already has everything synced, but now i wanted to move to windows. Where is the `blockchain_v1_mainnet.sqlite` i looked around a little but couldn't find it on linux.

Thanks

**Comment by LouisLoudogTrottier** [comment]
check in a hidden folder inside your home folder, mine was at 

ls ~/.chia/mainnet/db/

**Reply by hibenji** [reply]
Oh thanks, idk how i didn't come to that idea, i just completely missed the hidden folder thing.

**Comment by BrandtH22** [comment]
@hibenji , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
